src/main/java/org/broadinstitute/consent/http/resources/DACUserResource.java

```diff
@@ -63,11 +53,6 @@ public DACUserResource(VoteService voteService) {
     public Response createDACUser(@Context UriInfo info, String json) {
         try {
             DACUser dacUser = dacUserAPI.createDACUser(new DACUser(json));
-            if (isChairPerson(dacUser.getRoles())) {
-                dacUserAPI.updateExistentChairPersonToAlumni(dacUser.getDacUserId());
-                List<Election> elections = electionAPI.cancelOpenElectionAndReopen();
-                voteService.createVotesForElections(elections);
-            }
```

Comment:
This functionality is being replaced with https://broadinstitute.atlassian.net/browse/DUOS-491